libsolidity/ast/ASTVisitor.h

```diff
@@ -145,6 +145,7 @@ class ASTVisitor
 	virtual void endVisitNode(ASTNode&) { }
 };
 
+// Post visitor. endVisit ensure the child types are already visited
```

Comment:
Why `Post visitor`, or did you meant to write `Const visitor`? I don't understand the second sentence.